src/crypto/slow-hash.c

```diff
@@ -253,11 +267,18 @@ extern void aesb_pseudo_round(const uint8_t *in, uint8_t *out, const uint8_t *ex
 #define VARIANT4_RANDOM_MATH_INIT() \
   v4_reg r[9]; \
   struct V4_Instruction code[NUM_INSTRUCTIONS_MAX + 1]; \
+  int jit = use_v4_jit(); \
   do if (variant >= 4) \
   { \
     for (int i = 0; i < 4; ++i) \
       V4_REG_LOAD(r + i, (uint8_t*)(state.hs.w + 12) + sizeof(v4_reg) * i); \
     v4_random_math_init(code, height); \
+    if (jit) \
```

Comment:
Technically we don't need to generate code every time, only when height changes. Maybe add `hp_jitfunc_cur_height` here and check it?

Reply:
Sure, but that's safe and fast enough for now. I'll go and speed up lesser things at some point, like the number of TLS calls.